### PR TITLE
[F1] Using new Executor Format (with tokens and API URI) (5th)

### PR DESCRIPTION
**Pull request by stjohnjohnson**
[Feature](https://github.com/screwdriver-cd/screwdriver/issues/77)

Passing:
 - buildId - Build Identifier (to look up stuff)
 - container - Container to start in
 - apiUri - URI to hit our API
 - token - Token to be able to read/write from the API

executor `version` is the job-tools version to be used here https://github.com/screwdriver-cd/executor-k8s/blob/master/config/job.yaml.tim#L17

### CODE REVIEW

**Comment by d2lam** [comment]
test

File: package.json [hunk 1]
```diff
@@ -55,8 +55,8 @@
     "screwdriver-data-schema": "^7.0.0",
```
Comment (d2lam) [inline package.json]: 8.0?

**Comment by petey** [comment]
👍

**Comment by jer** [comment]
👍

**Pull request merged**